src/ImageSharp/Image.cs

```diff
@@ -131,11 +130,12 @@ public Image<TPixel2> CloneAs<TPixel2>()
             where TPixel2 : struct, IPixel<TPixel2>;
 
         /// <summary>
-        /// Accept a <see cref="IImageVisitor"/>.
+        /// Accepts a <see cref="IImageVisitor"/>.
         /// Implemented by <see cref="Image{TPixel}"/> invoking <see cref="IImageVisitor.Visit{TPixel}"/>
         /// with the pixel type of the image.
         /// </summary>
-        internal abstract void AcceptVisitor(IImageVisitor visitor);
+        /// <param name="visitor">The visitor.</param>
+        protected internal abstract void Accept(IImageVisitor visitor);
```

Comment:
It's unlikely that anyone will try to subclass `Image`, but in theory, we expose an extension point with `protected internal`, so `internal` is probably better.

Reply:
Ok, I've just seen #1011. See my comments there!

Reply:
Cool. Will update once I merge #1009